### PR TITLE
Mex fixes

### PR DESCRIPTION
**Pull request by acpaquette**
Fixes up MEX HRSC drivers for use in ISIS and USGSCSM

### CODE REVIEW

File: ale/drivers/mex_drivers.py [hunk 1]
```diff
@@ -69,7 +69,7 @@
             [-8569.5561557859, 0.068566503695773, 142.857126402363]],
 }
 
-class MexHrscPds3NaifSpiceDriver(LineScanner, Pds3Label, NaifSpice, RadialDistortion, Driver):
+class MexHrscPds3NaifSpiceDriver(LineScanner, Pds3Label, NaifSpice, NoDistortion, Driver):
```
Comment (Kelvinrr) [inline ale/drivers/mex_drivers.py:72]: why disable distortion?

File: tests/pytests/data/h5270_0000_ir2/h5270_0000_ir2_pds3.lbl [hunk 2]
```diff
@@ -115,147 +115,6 @@ RADIANCE_OFFSET              = 0.0 <W*m**-2*sr**-1>
 RADIANCE_SCALING_FACTOR      = 0.0169353 <W*m**-2*sr**-1>
 REFLECTANCE_SCALING_FACTOR   = 0.00215382
 
-/* DATA OBJECT DEFINITIONS */
-Object = IMAGE
-  INTERCHANGE_FORMAT = BINARY
-  LINES              = 400
-  LINE_PREFIX_BYTES  = 68
-  LINE_SAMPLES       = 1288
-  SAMPLE_TYPE        = MSB_INTEGER
```
Comment (Kelvinrr) [inline tests/pytests/data/h5270_0000_ir2/h5270_0000_ir2_pds3.lbl:124]: why is all of this deleted?

File: ale/drivers/mex_drivers.py (hunk 1 above)
Comment (acpaquette) [reply ale/drivers/mex_drivers.py:72]: I think this was before we had a `NoDistortion` class/mixin in ALE. In the driver we define the distortion coefficients as all zeros:
```
    @property
    def odtk(self):
        """
        The coefficients for the distortion model
        Returns
        -------
        : list
          Radial distortion coefficients. There is only one coefficient for LROC NAC l/r
        """
        return [0.0, 0.0, 0.0]
```

I just changed it to remove duplicate code but didn't remove the odtk property. I will do that

File: tests/pytests/data/h5270_0000_ir2/h5270_0000_ir2_pds3.lbl (hunk 2 above)
Comment (acpaquette) [reply tests/pytests/data/h5270_0000_ir2/h5270_0000_ir2_pds3.lbl:124]: For whatever reason two pds3 labels existed in the file. One with 400 lines and the other with 15088 lines which is the actual number of lines in the image.

Using the real number of lines made testing easier

**Review by Kelvinrr — APPROVED** [review]

**Pull request merged**